assets/js/recommender.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let NUM_RECOMMENDED_PAGES = 5
+
+let pageUrl = document.getElementById('full-page-url').innerHTML
+
+let param = {
+  url: unescape(window.btoa(pageUrl))
+}
+
+let request = $.ajax({
+  url: 'https://api.isomer.gov.sg/recommend',
+  data: param,
+  dataType: 'json'
+})
+
+request.then(function(response) {
+  let relatedPostsString = ''
+  let relatedPostsDiv = document.getElementById('related-content')
+  let relatedPostsList = document.getElementById('related-content-list')
+
+  relatedPostsDiv.classList.remove('hide')
```

Comment:
Possibly a little pedantic but would it be safer to place this line after `relatedPostsList.innerHTML` has been set just in case any exceptions crop up? :sweat_smile:

On that note it is probably a good idea to check for the "Internal server error" JSON response it returns when either a malformed base64 value is sent or the page isn't found in the related posts database

Reply:
With the `.done()` promise handler, we're guaranteed that the highlighted chunk of code will run only if the JSON response status is 200, so we're safe